components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -130,9 +151,11 @@ func (r *reconciler) reconcile(ctx context.Context, subscription *eventingv1alph
 		}
 	} else {
 		// The object is being deleted
+		KymaSubscriptionsGauge.DeleteKymaSubscriptionsGauge(subscription.Namespace, subscription.Name)
 		if util.ContainsString(&subscription.ObjectMeta.Finalizers, finalizerName) {
 			// our finalizer is present, so lets handle our external dependency
-			if err := r.deleteExternalDependency(ctx, knativeSubsName, knativeChannelName, knativeSubsNamespace); err != nil {
+			if err := r.deleteExternalDependency(ctx, knativeSubsName, knativeChannelName, knativeSubsNamespace,
+				subscription.Name, KnativeSubscriptionsGauge, KnativeChannelGauge); err != nil {
```

Comment:
Fix the formatting

Reply:
I believe it's correct and checked with the IDE